linux_pipeline/Jenkinsfile_azure_lis_hotfix_functional

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env groovy
+def FunctionalTests = """
+	NVIDIA-CUDA-DRIVER-VALIDATION-MAX-GPU,NVIDIA-GRID-DRIVER-VALIDATION-MAX-GPU,INFINIBAND-OPEN-MPI-2VM,NVME-MAX-DISK-VALIDATION,SRIOV-VERIFY-SINGLE-VF-CONNECTION-MAX-VCPU,
+	SRIOV-VERIFY-MAX-VF-CONNECTION-MAX-VCPU,SRIOV-RELOAD-MODULE,STORAGE-HOT-ADD-DISK-SERIAL,STORAGE-HOT-ADD-DISK-PARALLEL,VMBUS_VERIFY_INTERRUPTS,LIS-PREINSTALL-DISK-SIZE-VERIFICATION,
+	VERIFY-DPDK-BUILD-AND-TESTPMD-TEST,INFINIBAND-INTEL-MPI-2VM"""
+def DeployTests = "LIS-DRIVER-VERSION-CHECK,LIS-DEPLOY-SCENARIO-1,LIS-DEPLOY-SCENARIO-2,LIS-DEPLOY-SCENARIO-3,LIS-DEPLOY-SCENARIO-4,LIS-DEPLOY-SCENARIO-5,LIS-DEPLOY-SCENARIO-6,LIS-DEPLOY-SCENARIO-7,LIS-DEPLOY-SCENARIO-8"
+
+def RunPowershellCommand(psCmd) {
+	bat "powershell.exe -NonInteractive -ExecutionPolicy Bypass -Command \"[Console]::OutputEncoding=[System.Text.Encoding]::UTF8;$psCmd;EXIT \$global:LastExitCode\""
+}
+
+def CleanWorkspace() {
+	retry(5) {
+		cleanWs()
+	}
+}
+
+def Prepare() {
+	retry(5) {
+		cleanWs()
+		unstash 'LISAv2'
+	}
+}
+
+def ReportException(stagename, exc) {
+	def body = "<pre>"
+	body += "\nStage Name        : ${stagename}\n"
+	body += "\nException Message : ${exc}\n"
+	body += "\nBuild URL         : ${env.BUILD_URL}\n"
+	withCredentials([string(credentialsId: 'HOTFIX_DEV_MAIL', variable: 'HOTFIX_DEV_MAIL')]) {
+	emailext (
+		subject: "JOB: ${env.JOB_NAME} BUILD:${env.BUILD_NUMBER} Exception",
+		to: "${env.HOTFIX_DEV_MAIL}",
+		mimeType : "text/html",
+		body: body
+		)
+	}
+}
+
+stage ("Prerequisite") {
+	node ("meta_slave") {
+		cleanWs()
+		git branch: env.GIT_BRANCH, url: env.GIT_REPO
+		stash includes: '**', name: 'LISAv2'
+		cleanWs()
+	}
+}
+
+stageTimeout = 240
+def currentStage = "Deploy Tests"
+stage ("${currentStage}") {
+	def currentVHD = "${BASE_VHD}"
+	println "VHD for Deploy tests ${currentVHD}"
+	try {
+		timeout(time: stageTimeout, unit: 'MINUTES') {
+			node('azure') {
+				CleanWorkspace()
+				withCredentials([file(credentialsId: 'Azure_Secrets_TESTONLY_File', variable: 'Azure_Secrets_TESTONLY_File')]) {
+				Prepare()
+				println "Running LIS-DEPLOY_SCENARIOS"
```

Comment:
Remove this println, because, with previous println we can see, that we're running Deploy tests.

Reply:
removed println